operations/workplace/people/working-at-mattermost/paid-time-off.md

```diff
@@ -18,15 +18,13 @@ Note: If you are taking [Pregnancy](leaves-of-absence/pregnancy-leave.md) or [Pa
 
 ### When you take time-off, please do the following:
 
-1. Let your management know your [PTO](../../../../company/about-mattermost/list-of-terms.md#pto-or-paid-time-off) dates and who will be your [backup\(s\)](https://handbook.mattermost.com/operations/people/working-at-mattermost#5-be-the-change-you-want-to-see). 
-2. Add your time off to [BambooHR ](https://mattermost.bamboohr.com/)
+1. Let your management know your [PTO](../../../../company/about-mattermost/list-of-terms.md#pto-or-paid-time-off) dates and who will be your [backup\(s\)](https://handbook.mattermost.com/operations/people/working-at-mattermost#5-be-the-change-you-want-to-see).
+2. Post in the [Stand-up channel](https://community.mattermost.com/private-core/channels/stand-up)
 3. Enjoy your time off! 
```

Comment:
```suggestion
3. Block off time in the shared [Mattermost Time Off calendar](https://calendar.google.com/calendar/embed?src=mattermost.com_mg3nulgf6e70e50ohlq2rrkcnc%40group.calendar.google.com&ctz=America%2FLos_Angeles>), so it's easy for the team to see who is going to be away.
4. Enjoy your time off! 
```

Reply:
I think that calendar is read-only now so we'd have to update that back. Is that right @comharris ?

Reply:
Note that these suggested changes will only work if the Time Off calendar is made editable again (it is currently read-only).

Reply:
How about creating an event in your personal calendar and inviting the Time Off calendar instead of creating two? The Time Off calendar can be set to auto-accept all invites so no human intervention is required.

I see a few benefits to this:
1. We only need to create one event to block off time on your own calendar while concurrently indicating you will be off on the Time Off calendar for visibility.
2. Individuals can edit their own events while they remain ‘read-only’ for everyone else.
3. The Time Off calendar remains read-only, which prevents accidental editing of other peoples Time Off events while trying to create your own.